pupil-spa/src/app/services/pupil-prefs/pupil-prefs.service.ts

```diff
@@ -36,39 +36,38 @@ export class PupilPrefsService {
   }
 
   public async storePupilPrefs() {
+    if (!this.featureUseHpa) {
```

Comment:
Does the legacy mode still work?

Reply:
@jon-shipley we will need a tech work item to remove all of these checks once we are fully into HPA.